task/01-strings-tasks.js

```diff
@@ -130,7 +140,10 @@ function repeatString(value, count) {
  *   'ABABAB','BA' => 'ABAB'
  */
 function removeFirstOccurrences(str, value) {
-    throw new Error('Not implemented');
+    const startIndex = str.indexOf(value)
+    const endIndex = startIndex + value.length
+    const newStr = str.slice(0, startIndex).trim() + str.substring(endIndex)
+    return newStr
```

Comment:
Looks like a bit overengineered solution, there is also a `String` method that can help you to write a one-line solution, try to find it by yourself